unit_tests/test_config.py

```diff
@@ -637,6 +638,42 @@ def test_20_user_data_format_version_azure(self):
         # returns something from 5 months ago.
         self.assertIn('user_data_format_version', conf)
 
+    def test_21_verify_unique_stress_cmds_per_tenant(self):
```

Comment:
nitpick: this test isn't directly related to the configuration, i.e. `set_stress_command_to_tenant` isn't being called as part of the `SCTConfiguration()`, it can be tested on it's own test file.